### ISSUE TITLE
Project Overview

### ISSUE THREAD

**Issue by hmontero1205** [cmhavdala/intro-data-capstone-musclehub#1]
Overall, your work met the project requirements -- good job!

Your code is just on point and I have no issues to raise there.
Your project has all the content required but you are not conveying it in an effective manner. Tell a story when you present -- give context, show how you derive your results, give conclusions.

**Comment by cmhavdala** [comment]
Can you give me a couple of specific examples of how to tell a story?  I thought I gave conclusions.

**Comment by hmontero1205** [comment]
Yes but you jumped straight into the project without explaining why you were doing it or what musclehub is and what they were trying to find out. Also, you didn’t explain any of your methods

**Comment by cmhavdala** [comment]
So your saying I should explain each step in my presentation?  I thought that my audience would be bored and that they would be more interested in the final conclusion.  I tried to explain my methods on the first page after the title page.  But thanks for your comments!!
